clients/src/test/java/org/apache/kafka/clients/consumer/internals/ShareConsumerTestBuilder.java

```diff
@@ -152,7 +152,9 @@ public ShareConsumerTestBuilder(GroupInformation groupInfo, boolean enableAutoTi
                 subscriptions,
                 fetchConfig,
                 fetchBuffer,
-                metricsManager));
+                metricsManager,
+                retryBackoffMs,
+                DEFAULT_RETRY_BACKOFF_MAX_MS));
```

Comment:
I'd probably just use retryBackoffMs for the max parameter too. Just ensures that someone doesn't end up with the max smaller than the retryBackoffMs in some weird situation.